content/en/docs/v3/guides/benefits.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Benefits
+linktitle: Benefits
+description: Benefits of the new helm 3 based boot
+weight: 80
+---
+
+
+* We can use vanilla helm 3 now to install, update or delete charts in any namespace without needing tiller or custom code to manage `helm template`
+  * We can avoid all the complexities of the `jx step helm apply` logic using our own helm template generation + post processing logic. We can also move away from boot's use of `{{ .Requirements.foo }}` and `{{ .Parameters.bar }}` expressions
+* It opens the door to a flexible [multi-cluster support](/docs/v3/guides/multi-cluster/) so that every cluster/environment can be managed in the same canonical GitOps approach (as each cluster can use `jx boot` whether its a dev environment or remote staging/production environment)
+* We can use the `helm list` command line to view versions of each chart/app nicely in the CLI.
+  * we can avoid composite charts to simplfiy configuration and upgrades
+* Everything is now an app. So if you want to remove our `nginx-ingress` chart and replace it with another ingress solution (knative / istio / gloo / ambassador / linkerd or whatever) just go ahead and use the [apps commands](/docs/v3/guides/apps/) to add/remove apps and have boot manage everything in a consistent way
+    * e.g. here's [an example](https://github.com/jstrachan/environment-bucketrepo-dev/blob/master/helmfile.yaml#L2-L5) of removing `nginx`, `chartmusem` and `nexus` and adding in `istio`, `flagger` and some other tools like `bucketrepo` and `kuberhealthy` via a single simple yaml change. Incidentally to replace the use of `Ingress` resources with istio's `Gateway` and `VirtualService` you also need to add the `kind: istio` flag in the [jx-requirements.yml](https://github.com/jstrachan/environment-bucketrepo-dev/blob/master/jx-requirements.yml#L57) file
```

Comment:
Will this repo move to the jx (or similar) org?

Reply:
ah - let me switch to the jenkins x equivalent - great catch!